### PR TITLE
Enigma, Rocketman, Trappist: update nightlies

### PR DESCRIPTION
**Pull request by cgcgcg**
@trilinos/muelu

## Motivation
Implement similar changes as on geminga.

### CODE REVIEW

File: cmake/ctest/drivers/rocketman/cron_driver.sh [hunk 1]
```diff
@@ -35,21 +35,20 @@ export TDD_HTTPS_PROXY=$https_proxy
...
-module load sems-env
-module load sems-git
-module load sems-cmake/3.17.1
-module load sems-gcc/5.3.0
-module load sems-openmpi/1.10.1
-module load sems-superlu/4.3/base
-module load sems-hdf5/1.8.12/parallel
-module load sems-netcdf/4.4.1/exo_parallel
-module load sems-boost/1.63.0/base
+module load muelu-gcc
```
Comment (jhux2) [inline cmake/ctest/drivers/rocketman/cron_driver.sh:50]: Do I need to create a custom module for `muelu-gcc`?

File: cmake/ctest/drivers/enigma/cron_driver.sh [hunk 2]
```diff
@@ -35,37 +35,39 @@ export TDD_HTTP_PROXY=$http_proxy
...
 
-# load OpenMPI 1.6.4 (standard in RHEL 7)
-#module load mpi/openmpi-x86_64
-module load sems-env
-module load sems-cmake/3.17.1
-module load sems-gcc/5.3.0
-module load sems-openmpi/1.10.1
-module load sems-superlu/4.3
-
+module load muelu-gcc
```
Comment (jhux2) [inline cmake/ctest/drivers/enigma/cron_driver.sh:54]: Do I need to create a custom module file for `muelu-gcc`?

File: cmake/ctest/drivers/rocketman/cron_driver.sh (hunk 1 above)
Comment (cgcgcg) [reply cmake/ctest/drivers/rocketman/cron_driver.sh:50]: I added the module file

**Review by jhux2 — DISMISSED** [review]
Zehr gut!

**Review by jhux2 — DISMISSED** [review]
lgtm

**Review by jhux2 — APPROVED** [review]
yes!

**Pull request merged**